tests/integration-tests/Cargo.toml

```diff
@@ -74,6 +74,7 @@ dns-over-h3 = [
 # dns-over-native-tls = ["dns-over-tls",  "hickory-resolver/dns-over-native-tls", "hickory-server/dns-over-native-tls"]
 dns-over-openssl = [
     "dns-over-tls",
+    "dnssec-openssl",
```

Comment:
This is necessary because `hickory-server/dns-over-openssl` enables `hickory-server/dnssec-openssl` and `hickory-server/dnssec`. The state of `hickory-integration/dnssec` and `hickory-server/dnssec` need to match, because `#[cfg(feature = "dnssec")]` modifies both the definition of the `Authority` trait in one crate, and the implementation of that trait on `TestAuthority` in the other.